src/olympia/addons/admin.py

```diff
@@ -298,7 +297,7 @@ def wrapper(*args, **kwargs):
 
         urlpatterns = super(AddonAdmin, self).get_urls()
         custom_urlpatterns = [
-            url(
+            re_path(
```

Comment:
This isn't just a replacement, right? `url` and `re_path` don't seem to be equivalent.

Reply:
https://docs.djangoproject.com/en/2.2/ref/urls/#django.conf.urls.url